### ISSUE TITLE
Add 'Remove Follower' Option for Users

### ISSUE THREAD

**Issue by Brayozin** [bluesky-social/social-app#5068]
**Is your feature request related to a problem? Please describe.**
No. It's an feature request

Currently, users cannot remove followers directly. The only option available is to block the follower, but once the user unblocks them, the follower remains.

**Describe the solution you'd like**
Introduce a feature that allows users to remove a follower directly.

I would like users to have the ability to remove a follower without needing to block them.

**Describe alternatives you've considered**
Blocking and then unblocking the follower, but this method is ineffective as the follower remains after unblocking.

Blocking and unblocking the follower, but this approach does not solve the issue.

**Additional context**

**Comment by Brayozin** [comment]
duplicate of #1160
